eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -159,12 +159,12 @@ private ISubmittedVariant createVariantWithContextBase(ISubmittedVariant normali
             // VCF 4.2 section 1.4.1.4. REF: "the REF and ALT Strings must include the base before the event unless the
             // event occurs at position 1 on the contig in which case it must include the base after the event"
             newStart = normalizedVariant.getStart() + 1;
-            contextBase = fastaSequenceReader.getSequence(normalizedVariant.getContig(), newStart, newStart);
+            contextBase = fastaSequenceReader.getSequenceToUpperCase(normalizedVariant.getContig(), newStart, newStart);
```

Comment:
Given that the base is going to be added to the VCF, I think we should reflect what is in the FASTA sequence. VCF accepts any allele capitalization.